recipes/seqverify/meta.yaml

```diff
@@ -1,25 +1,38 @@
-{% set version = "0.1.0" %}
+{% set version = "1.0.0" %}
 
 package:
   name: seqverify
   version: {{ version }}
 
 source:
-  url: https://github.com/mpiersonsmela/SeqVerify/archive/refs/tags/prerelease.tar.gz
-  sha256: 75d479d485a27e4072ac272fd0f690a7b557bdb56a27ddd377736823e5297b94
+  url: https://github.com/mpiersonsmela/SeqVerify/archive/refs/tags/v1.0.0.tar.gz
+  sha256: 2db1194cdb51e9252936cf5b60e702cec3f170bc6ceeb7b3371745bddd4f39fc
 
 build:
   noarch: generic
   number: 0
+  run_exports:
+    - {{ pin_subpackage('seqverify', max_pin="x.x") }}

```

Comment:
See PR template: in such a case (max_pin=x.x with version >=1.0.0), please add a note (as a comment) that shortly mentions your evidence for the need of x.x. Otherwise, use max_pin="x".

Reply:
Upon reviewing the code we figured max_pin="x" would work fine, actually, so we reverted to it. Thanks!